openshift/manifests/18-deployment-openshift-operator-controller-operator-controller-controller-manager.yml

```diff
@@ -122,4 +125,8 @@ spec:
             path: /etc/containers
             type: Directory
           name: etc-containers
+        - hostPath:
+            path: /var/lib/kubelet
+            type: Directory
+          name: auth-from-host
```

Comment:
```suggestion
            path: /var/lib/kubelet/config.json
            type: File
          name: global-auth-file
```

Reply:
This is a generated file, so the source will have to be modified.

Reply:
Moved my suggestion to the source of this change.